app/views/equipment_items/show.html.erb

```diff
@@ -33,4 +33,3 @@
 <hr>
 <%= link_to "Edit", edit_equipment_item_path(@equipment_item), class: 'btn btn-default' %>
 <%= equipment_item.make_deactivate_btn %>
-<%= link_to "View All", equipment_items_path, class: 'btn btn-default' %>
```

Comment:
same here, we can just go to the list of equipment items for the item's equipment model.